<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Mid-Level Sensor Viewer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(86, 87, 91);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>5</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(84, 40, 72);
	font: 75 40pt &quot;Times New Roman&quot;;
	color: rgb(255, 255, 255);
	border: 1px solid white;
	border-radius: 10px;
	text-align: center;
}

QPushButton::hover {
	background-color: rgb(38, 201, 84);
	color: rgb(255, 255, 255);
    border: 0px;
	border-radius: 10px;
	text-align: center;
}</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 157, 225);
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>5</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(84, 40, 72);
	font: 75 40pt &quot;Times New Roman&quot;;
	color: rgb(255, 255, 255);
	border: 1px solid white;
	border-radius: 10px;
	text-align: center;
}

QPushButton::hover {
	background-color: rgb(38, 201, 84);
	color: rgb(255, 255, 255);
    border: 0px;
	border-radius: 10px;
	text-align: center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/images/images/system_img.png</selectedon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/images/images/sensor_error.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>85</y>
      <width>585</width>
      <height>395</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(84, 40, 72);
border: 1 px solid white;
background-repeat; no-repeat;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>495</width>
       <height>79</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 40, 72);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>79</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato-Bold</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;
font: 24px 'Lato-Bold';</string>
     </property>
     <property name="text">
      <string>COMPARISON :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>125</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato-Bold</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;
font: 24px 'Lato-Bold';</string>
     </property>
     <property name="text">
      <string>TEST :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>190</y>
       <width>80</width>
       <height>70</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 10px;
	color: rgb(0, 0, 0);
    background-color: rgb(225, 225, 225);
	text-align: center;
	font: 44px 'Lato-Bold';
}

QPushButton::hover {
	color: rgb(255, 255, 255);	
    background-color: rgb(38, 201, 84);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>290</y>
       <width>80</width>
       <height>70</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 10px;
	color: rgb(0, 0, 0);
    background-color: rgb(225, 225, 225);
	text-align: center;
	font: 44px 'Lato-Bold';
}

QPushButton::hover {
	color: rgb(255, 255, 255);	
    background-color: rgb(38, 201, 84);
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>160</y>
       <width>495</width>
       <height>79</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 40, 72);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>79</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>234</y>
       <width>495</width>
       <height>79</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 40, 72);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>79</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>308</y>
       <width>299</width>
       <height>79</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 40, 72);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>79</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>308</y>
       <width>201</width>
       <height>79</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84, 40, 72);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>79</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>190</y>
       <width>80</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>260</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato-Bold</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;
font: 24px 'Lato-Bold';</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>95</y>
       <width>80</width>
       <height>70</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid white;
	border-radius: 10px;
	color: rgb(0, 0, 0);
    background-color: rgb(225, 225, 225);
	background-image: url(:/images/images/clear_open_img.png);
}

QPushButton::hover {
	color: rgb(255, 255, 255);	
    background-color: rgb(38, 201, 84);
    background-image: url(:/images/images/clear_close_img.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <zorder>scrollArea</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>scrollArea_2</zorder>
    <zorder>scrollArea_3</zorder>
    <zorder>scrollArea_4</zorder>
    <zorder>scrollArea_5</zorder>
    <zorder>label_12</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>label_11</zorder>
    <zorder>pushButton_3</zorder>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>Sensor Viewer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>85</y>
      <width>210</width>
      <height>395</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(84, 40, 72);
border: 1 px solid white;
background-repeat; no-repeat;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>190</width>
       <height>80</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 3px solid black;
	border-radius: 5px;
	color: rgb(0, 0, 0);
    background-color: rgb(225, 225, 225);
	text-align: center;
	font: 26px 'Lato-Bold';
}

QPushButton::hover {
	color: rgb(255, 255, 255);	
    background-color: rgb(38, 201, 84);
}</string>
     </property>
     <property name="text">
      <string>COMPARISON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>190</width>
       <height>100</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 3px solid black;
	border-radius: 5px;
	color: rgb(0, 0, 0);
    background-color: rgb(225, 225, 225);
	text-align: center;
	font: 36px 'Lato-Bold';
}

QPushButton::hover {
	color: rgb(255, 255, 255);	
    background-color: rgb(38, 201, 84);
}</string>
     </property>
     <property name="text">
      <string>TEST</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato-Bold</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 255);
color: rgb(255, 255, 255);
border: 0px;
font: 36px 'Lato-Bold';</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato-Bold</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 255);
color: rgb(255, 255, 255);
border: 0px;
font: 36px 'Lato-Bold';</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>190</width>
       <height>80</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 3px solid black;
	border-radius: 5px;
	color: rgb(0, 0, 0);
    background-color: rgb(225, 225, 225);
	text-align: center;
	font: 26px 'Lato-Bold';
}

QPushButton::hover {
	color: rgb(255, 255, 255);	
    background-color: rgb(38, 201, 84);
}</string>
     </property>
     <property name="text">
      <string>STANDARD</string>
     </property>
    </widget>
    <zorder>pushButton_7</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>label_14</zorder>
    <zorder>label_13</zorder>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>SENSOR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>250</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>TEXT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 3, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>250</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0%);
color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
